components/law-finops-dashboard/variables.tf

```diff
@@ -0,0 +1 @@
+../../environments/variables.tf
```

Comment:
is that symlink correctly?

Reply:
I beleive it does, or else this dashboard would have never deployed. https://portal.azure.com/#@hmcts.net/dashboard/arm/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c/resourcegroups/law-finops-dashboard-test-rg/providers/microsoft.portal/dashboards/hmcts-qa-law-finops-dashboard

Reply:
ty